packages/legacy/core/App/modules/openid/utils/utils.tsx

```diff
@@ -17,11 +18,11 @@ export function sanitizeString(str: string) {
 }
 
 export function getHostNameFromUrl(url: string) {
-  //TODO: Find more elegant way to extract host name
-  // const urlRegex = /^(.*:)\/\/([A-Za-z0-9-.]+)(:[0-9]+)?(.*)$/
-  // const parts = urlRegex.exec(url)
-  // return parts ? parts[2] : undefined
-  return url.split("https://")[1]
+  try {
+    return getDomainFromUrl(url)
+  } catch (error) {
+    return 'undefined!'
```

Comment:
This string is kind of a strange return for an error. Wouldn't `null` or an actual `undefined` be better?

Reply:
done